src/diepvries/deserializers/snowflake_deserializer.py

```diff
@@ -29,9 +29,19 @@ class DatabaseConfiguration:
 
     database: str
     user: str
-    password: str
     warehouse: str
     account: str
+    password: Optional[str] = None
```

Comment:
Would be nice to eventually move away from `Optional` to the built-in `|` (or) operator

Reply:
Only from Python 3.10, and 3.8 and 3.9 are still supported by diepvries ATM. Although I'd be in favor of moving to newer Pythons ;)

Reply:
Aha that's an important point 🙃